charts/pxc-operator/README.md

```diff
@@ -38,6 +38,8 @@ The chart can be customized using the following configurable parameters:
 | `logStructured`                 | Force PXC operator to print JSON-wrapped log messages                                          | `false`                                          |
 | `logLevel`                      | PXC Operator logging level                                                                     | `INFO`                                           |
 | `disableTelemetry`              | Disable sending PXC Operator telemetry data to Percona                                         | `false`                                          |
+| `watchNamespace`                | Set when a different from default namespace is needed to watch (comma separated if multiple needed)  | `""`               |
+| `createNamespace`               | Set if you want to create watched namespaces with helm                                         | `false`                                   |
```

Comment:
I would like to suggest a different wording here:

Create the watched namespace(s)